poc/find_wr_params.sage

```diff
@@ -391,32 +391,41 @@ def display_vdaf_params(vdaf_params):
     print(printed_output)
 
 
+class Field32(Field):
+    """A fake 32-bit finite field. """
+
+    MODULUS = (2^32).previous_prime()  # May not be FFT-friendly.
```

Comment:
We could use [this field](https://github.com/divviup/libprio-rs/blob/bc8c3ec5feed9c6f68f113d148eff2788354d346/src/fp.rs#L346), which is known to be FFT-friendly. (It's the field used by Prio2.)